### ISSUE TITLE
English and French maps do not share a configuration unless forced

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/storylines-editor#447]
## Description

The UX improvements to edit English and French slides in parallel are extremely good. But the workflow for maps is confusing when the RAMP configuration file should be the same in both English and French. 

### Is your feature request related to a problem?
If you create a map in English, and then work on the same slide in French, creating a new map writes a separate configuration file instead of using the previously created on. 
![image](https://github.com/user-attachments/assets/cfccad69-a531-43af-9f6e-ab11d82d6bc6)

## Proposed solution
It would be nice if there was logic to see there is already a RAMP config in the English version of the slide, and append the French configuration in that, instead of creating two different files for two different languages.

**Comment by gordlin** [comment]
On hold until #414 (shared assets folder) is completed. It would be best if the shared map configurations integrates with that.
